### PR TITLE
Create hello.sh

### PR DESCRIPTION
**Pull request by kylinlizhun**

### CODE REVIEW

File: hello.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	"strings"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+	"k8s.io/component-base/logs"
+	logsapi "k8s.io/component-base/logs/api/v1"
+	_ "k8s.io/component-base/logs/json/register"
+	"k8s.io/kubernetes/test/utils/ktesting"
+)
```
Comment (kylinlizhun) [inline hello.sh:29]: good job

**Review by freelizhun — COMMENTED** [review]
dddd

File: hello.sh (hunk 1 above)
Comment (kylinlizhun) [reply hello.sh:29]: dddd

Comment (freelizhun) [reply hello.sh:29]: 需要kylinlizhun 在Files changed处点即submit reviews才能被freelizhun或者所有人看见，不然此处的评论会一直处于pending状态

File: hello.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	// scheduler_perf test we want more control over the flags, therefore
+	// here strip them out.
+	var fs flag.FlagSet
+	flag.CommandLine.VisitAll(func(f *flag.Flag) {
+		switch f.Name {
+		case "log-flush-frequency", "v", "vmodule":
+			// These will be added below ourselves, don't copy.
+		default:
+			fs.Var(f.Value, f.Name, f.Usage)
+		}
```
Comment (freelizhun) [inline hello.sh:46]: this want means?

Comment (kylinlizhun) [reply hello.sh:46]: no means

**Pull request merged**